manifests/y/YACReader/YACReader/9.13.1/YACReader.YACReader.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: YACReader.YACReader
+PackageVersion: 9.13.1
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: YACReader_is1
+ReleaseDate: 2023-07-16
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/YACReader/yacreader/releases/download/9.13.1/YACReader-v9.13.1.2307166-winx86-7z.exe
+  InstallerSha256: 2D951763C7DC9447FD64B02C565A2445FD38EAB63913B7615557A2A8312B2C8E
+- Architecture: x64
+  InstallerUrl: https://github.com/YACReader/yacreader/releases/download/9.13.1/YACReader-v9.13.1.2307166-winx86-7z.exe
+  InstallerSha256: 2D951763C7DC9447FD64B02C565A2445FD38EAB63913B7615557A2A8312B2C8E

```

Installer mismatch with `x64`;
```suggestion
  InstallerUrl: https://github.com/YACReader/yacreader/releases/download/9.13.1/YACReader-v9.13.1.2307166-winx64-7z-qt6.exe
  InstallerSha256: 80AEF50BE7CB00D2F2787AA5D1F2FB47242A06F406E5D6D6DF34908C9C83A15A
```